### ISSUE TITLE
Writting at wrong Histogram Column

### ISSUE THREAD

**Issue by Markets00** [ocornut/imgui#1770]
**Version/Branch of Dear ImGui:**
1.53 with SFML, using C++11

**My Issue/Question:** 
I'm trying to fill a Histogram from an InputText, but the index which I modify from the floats vector, is  rendered in a different column (index) at the histogram.

**Code example:**
```
// Global variables
std::vector<int> intVector = std::vector<int>(100, 15);
std::string debugStr    = "";

// Local variables and code
std::string subStr    = "";
static char buffer[100];
float values[100];
for(int i = 0; i < 100; i++)
    values[i] = (float)intVector[i];

ImGui::PlotHistogram("Histogram", values, IM_ARRAYSIZE(values), 20, NULL, 0, 15, ImVec2(400,80));

if(ImGui::InputText("##histInput", buffer, IM_ARRAYSIZE(buffer))) {
    std::string strBuffer = buffer;
    std::istringstream iss(strBuffer);

    int idx = 0;
    debugStr = "";
    while(iss && idx < 100) {
        subStr = "";
        iss >> subStr;
        if(!subStr.empty()) {
            debugStr = debugStr + "[" + std::to_string(idx) + "," + std::to_string(std::stoi(subStr, nullptr)) + "]: " + std::to_string((float)intVector[idx]) + " ";
            intVector[idx++] = std::stoi(subStr, nullptr);
        }
    }
}
ImGui::TextUnformatted(debugStr.c_str());
```

**Screenshot**
![image](https://user-images.githubusercontent.com/20399842/39146525-4bb73b90-4737-11e8-86c7-b4c4ad716b23.png)
As you can see, the info displayed with debugStr is: [target index, input value]: intVector value.
The first time you modify the text, it shows you the vector stored value, but after the first time, it shows the right value which it have to store in intVector, and copy to the values (float*).

Thank you!

**Comment by ocornut** [comment]
Your code and the screenshot don't seem match which suggest that you haven't tested the code you posted.
Your code uses `values_count = 100` and the screenshot shows 30 values.
Your code uses `values_offset = 20` and the screenshot shows an offset of 12.

You modified parameters before writing the message, yet if you looked at those parameters you'll probably understand the error.

**Comment by Markets00** [comment]
> Sorry for being so blunt so I've been receiving way too many questions to handle.

Don't worry, it's obvious and you are doing it right. The screenshot is from my working project, and the code is an isolation of the bug, that's why code and screenshot didn't match.
I'm used to debug my own code, and look for solutions through documentations and FAQs in order to not waste time, specially, other's time. That's the reason why I'm not used to post issues ^^', I'm sorry about that.

Anyway, you where right, I was misunderstanding values_offset parameter, and setting it to 0 worked perfectly, but in the screenshot which you thought that values_offset was 12, it was 20, as the example code I posted (I guess I copied that value from another issue or example).

Thanks again, and go ahead!
